### PR TITLE
silvia/use-cases: suggestions in bulk

### PR DESCRIPTION
**Pull request by wesleyboar**
## Overview

If [suggestions through GitHub](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request) are not a suitable medium for the contributor.

## Related

- all my suggestions for #37

## Changes

- **removed** extraneous `<br/>`
- **added** new lines before and after lists
- **fixed** a broken link

### `user-guide/docs/usecases/padgett/usecase_JN_viz.md`

| with `<br/>`<br><sup>has no effect</sup> | sans `<br/>`<br><sup>renders the same</sup> |
| - | - |
| usecase_JN_viz with br (has no effect) | usecase_JN_viz sans br (renders the same) |

### `user-guide/docs/usecases/vantassel_and_zhang/usecase.md`

| before | after |
| - | - |
| vantassel_and_zhang BEFORE | vantassel_and_zhang AFTER |

### CODE REVIEW

**Comment by wesleyboar** [comment]
@silviamazzoni, if you would rather see all my suggestions and testing for #37 in bulk, see this PR.

**Comment by silviamazzoni** [comment]
your fixes are beautiful.

**Comment by silviamazzoni** [comment]
what does "Squash and merge" mean? it's an option in the green button

**Comment by wesleyboar** [comment]
**TL;DR:** You'll see a commit "**silvia/use-cases: suggestions in bulk (#40)**" in [your branch's commits](https://github.com/DesignSafe-CI/DS-User-Guide/pull/37/commits) instead of "**silvia/use-cases: suggestions in bulk […]**" (and other commits, had I made any) from [this branch's commits](https://github.com/DesignSafe-CI/DS-User-Guide/pull/40/commits).

_It is moot merge behavior for this PR, because I only made one commit._

<details>

All commits in this branch `silvia/use-cases--suggestions-in-bulk` are merged into the target branch `silvia/use-cases` as a single commit. This makes it so that commits in `silvia/use-cases` are not "polluted" by extraneous history of all the commits in `silvia/use-cases--suggestions-in-bulk`.

You may read GitHub's article on the feature, "[Squashing your merge commits](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/about-merge-methods-on-github#squashing-your-merge-commits)".

</details>
